Studio-Lovelies/StudioLoveliesWebsite
# Contact form message uses relative timestamps inside Discord

## Summary
Uses [Discord timestamps](https://gist.github.com/LeviSnoot/d9147767abeef2f770e9ddcd91eb85aa) to always show the time of messages relative to the user's timezone.

Also relocates the Pull Request template to the proper directory. (accidentally moved it to the workflows directory in #2)

## Before/after screenshots and/or animated gif
- Before:
![Stuck to GMT](https://github.com/Studio-Lovelies/StudioLoveliesWebsite/assets/1689033/9d70677f-419e-4cdf-aed9-28e30b72a83a)
- After:
![Relative to the user's active timezone](https://github.com/Studio-Lovelies/StudioLoveliesWebsite/assets/1689033/2708d50f-8325-4659-83cc-d379606e8944)


## Testing instructions
1. Verify https://preview.studio-lovelies.com/version is on the latest commit of this Pull Request
2. Visit https://preview.studio-lovelies.com/contact
3. Send a message
4. Check the internal Discord server for a new message and verify the timestamp is relative to your local timezone

## Additional information

- `[ ]` Changes UI
- `[ ]` Has associated resource:
